### PR TITLE
Add search by keyword and portion of quote command

### PR DESCRIPTION
**Pull request by shikhir-arora**
Add search quote command that returns a quote matching a keyword _and_ a string portion of text from a quote within that keyword's quotes.

Example: keyword `test` contains 3 quotes: `hello`, `how are you?`, and `not too bad`

Search quote for 'are': `.qsearch test are` (usage: `{prefix}qsearch keyword text`) returns `how are you?` in this case.

### CODE REVIEW

**Comment by samdivaio** [comment]
Looks nice, I really appreciate your PR and all the time you spent on it for us.
I however looking for something like listing all the quotes saved with a single keyword, that'd be even more awesome.

**Comment by shikhir-arora** [comment]
@samdivaio I will work on that, won't be hard to do at all in an embed list. 😄 

The search command above is useful in servers that use quotes a lot, for example we have a **ton** of quotes saved, the majority are just saved as `@username` for the keyword. Many times a user wants to pull up a quote they know was funny, etc. in conversation or context. So it comes in handy i.e. `.qsearch @username sometext` expands on the random quote for keyword, since some have a ton of quotes.

**Comment by samdivaio** [comment]
Your PR is already awesome and useful no doubt there :D

**Comment by shikhir-arora** [comment]
img1

<br><br>
img2 

Just some examples of how it works on our server (we use `.qadd` and `.qshow` instead of `..` or `...` since it gets confusing and also display the keyword with quotes) 😉

**Comment by shikhir-arora** [comment]
ignore the last 3, i'm an idiot >.<

**Comment by shikhir-arora** [comment]
Ignore the reverts, i'm an idiot. >.<

PR should be up to date with dev as of this message

File: src/NadekoBot/Resources/CommandStrings.resx [hunk 1]
```diff
@@ -1143,6 +1143,15 @@
   <data name="showquote_usage" xml:space="preserve">
     <value>`{0}.. abc`</value>
   </data>
+  <data name="searchquote_cmd" xml:space="preserve">
+    <value>qsearch ...</value>
```
Comment (Kwoth) [inline src/NadekoBot/Resources/CommandStrings.resx]: just remove ... alias

Comment (shikhir-arora) [inline src/NadekoBot/Resources/CommandStrings.resx]: sure thing

**Pull request merged**